lib/model/Fixture.js

```diff
@@ -0,0 +1,173 @@
+const path = require('path');
+const fs = require('fs');
+
+const Meta = require(path.join(__dirname, 'Meta.js'));
+const Physical = require(path.join(__dirname, 'Physical.js'));
+const Channel = require(path.join(__dirname, 'Channel.js'));
+const Mode = require(path.join(__dirname, 'Mode.js'));
+
+module.exports = class Fixture {
+  constructor(jsonObject) {
+    this.jsonObject = jsonObject; // calls the setter
+  }
+
+  set jsonObject(jsonObject) {
+    this._jsonObject = jsonObject;
+    this._cache = {};
+  }
+
+
+  get name() {
+    return this._jsonObject.name; // required
+  }
+
+  get shortName() {
+    return this._jsonObject.shortName || this._jsonObject.name;
+  }
+
+  get categories() {
+    return this._jsonObject.categories; // required
+  }
+
+  get mainCategory() {
+    return this.categories[0];
+  }
+
+  get meta() {
+    if (!('meta' in this._cache)) {
+      this._cache.meta = new Meta(this._jsonObject.meta);
+    }
+
+    return this._cache.meta;
+  }
+
+  get comment() {
+    return this._jsonObject.comment || '';
+  }
+
+  get hasComment() {
+    return 'comment' in this._jsonObject;
+  }
+
+  get manualURL() {
+    return this._jsonObject.manualURL || null;
+  }
+
+  /*
+    benchmark results (10,000,000 iterations):
```

Comment:
Please move the comment to above the class and inline the benchmark code (just the relevant parts), since we use this caching technique in the whole model structure.